hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/metrics/RBFMetrics.java

```diff
@@ -371,13 +372,13 @@ private static void setStateStoreVersions(
   }
 
   @Override
-  public long getTotalCapacity() {
-    return getNameserviceAggregatedLong(MembershipStats::getTotalSpace);
+  public BigInteger getTotalCapacity() {
```

Comment:
Not sure the change of public API is proper or not.

Reply:
According to the [document](http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/Compatibility.html#Metrics.2FJMX), the change is incompatible. However, I think we can do the incompatible change to fix this issue.

Reply:
Another way to do this is adding new metrics in BigInt instead of overwriting exiting one. We can call them getTotalCapacityBigInt() What do you think?

Reply:
Adding new metrics is okay to me. Thanks.